### PR TITLE
feat: adding Buddy's Pizza 

### PR DESCRIPTION
**Pull request by chrisdemars**
## Description

This PR adds Buddy's Rendezvou to the regional pizza spots.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Related to #35 

## Mobile & Desktop Screenshots/Recordings

n/a

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
n/a

### CODE REVIEW

File: regional-pizza.md [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
 
-* Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
-* Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
-* Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
-Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
+- Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
+- Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
+- Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
+  Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
```
Comment (CBID2) [inline regional-pizza.md:26]: Please revert theses

Comment (CBID2) [inline regional-pizza.md:26]: Please refer to [my comment in the next line](https://github.com/open-sauced/pizza-verse/pull/44/files#r1366235972) for context on how to improve this section.
```suggestion
- Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
- Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
- Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
  Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
```

File: regional-pizza.md [hunk 2]
```diff
@@ -1,43 +1,62 @@
...
+
+> The style flips traditional pizza on its head by beginning with double proofed dough perfectly fit to a square pan – a process that allows the dough to rise and be stretched twice – followed by the backwards layering of toppings.
+>
+> [USA Today](https://www.usatoday.com/story/travel/columnist/greatamericanbites/2018/10/17/pizza-regional-styles/1666339002/) called Buddy’s Pizza “Pilgrimage-worthy pizza” and [The Daily Meal](https://www.thedailymeal.com/eat/101-best-pizzas-america-2020/slide-97) named it #6 of the 101 Best Pizzas in America.
+>
+> The pepperoni is placed directly on the dough allowing the rich flavors of the meat to be absorbed into the crust. Next, crumbled Wisconsin brick cheese – a proprietary blend created just for Buddy’s – is sprinkled across the pie before the other toppings. Finally, to finish off the unique look, three wide stripes of sauce are skimmed across the top of the pizza before being baked to perfection.
+
+Buddy's is a staple in the Detroit community when it comes to pizza!
+
+<img src="./images/six-mile.jpg" width="300" alt="Buddy's Rendezvous on six mile and conant.">
```
Comment (CBID2) [inline regional-pizza.md:56]: Consider placing your image underneath the title instead of here.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @chrisdemars! :) Welcome to OpenSauced! :) Your entry is off to a great start. It just needs a few tweaks.

File: regional-pizza.md (hunk 1 above)
Comment (chrisdemars) [reply regional-pizza.md]: Prettier did all that lol. I disabled Prettier and switched it back.

**Comment by chrisdemars** [comment]
@CBID2 all should be fixed

**Review by CBID2 — APPROVED** [review]
Now moving on to more pressing measures...
1. Detroit Pizza sounds awesome! :) 
2. Let's merge this PR!! :)  
3. A little tip: Always fill out every section of the pull request form. It helps maintainers and other contributors understand how to review your contribution. For more tips, check out the blog post, [Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9).

**Comment by chrisdemars** [comment]
@CBID2 I wanted to submit another PR for a different pizza place but I keep getting a `permission denied` in my terminal.

**Comment by CBID2** [comment]
> @CBID2 I wanted to submit another PR for a different pizza place but I keep getting a `permission denied` in my terminal.

Hi @chrisdemars! 😊 I'm discussing it with the community team, but I think it's because we're trying to keep it one PR a minimum.

**Comment by chrisdemars** [comment]
> > @CBID2 I wanted to submit another PR for a different pizza place but I keep getting a `permission denied` in my terminal.
> 
> Hi @chrisdemars! 😊 I'm discussing it with the community team, but I think it's because we're trying to keep it one PR a minimum.

Oh that is a bummer lol. I was going to get my 4 PR's for Hacktoberfest, all pizza related stuff I could contribute.

**Comment by CBID2** [comment]
> > > @CBID2 I wanted to submit another PR for a different pizza place but I keep getting a `permission denied` in my terminal.
> > 
> > 
> > Hi @chrisdemars! 😊 I'm discussing it with the community team, but I think it's because we're trying to keep it one PR a minimum.
> 
> Oh that is a bummer lol. I was going to get my 4 PR's for Hacktoberfest, all pizza related stuff I could contribute.

Aww it'll be ok @chrisdemars. I'll give you an update on what the team says. In the meantime, check out [this section of our Hacktoberfest page](https://docs.opensauced.pizza/community/hacktoberfest/#im-ready-for-hacktoberfest).

**Pull request merged**